data/test/scenarios/wml_tests/UnitsWML/Attacks/attack_alignment.cfg

```diff
@@ -0,0 +1,24 @@
+#textdomain wesnoth-test
+
+{COMMON_KEEP_A_B_UNIT_TEST "attack_alignment_test" (
+    [event]
+        name=start
+        [modify_unit]
+            [filter]
+            [/filter]
+            [effect]
+                apply_to = new_ability
+                [abilities]
+                    {ABILITY_ILLUMINATES}
+                [/abilities]
+            [/effect]
+            [effect]
+                apply_to=attack
+                set_attack_alignment=lawful
+            [/effect]
+        [/modify_unit]
+
+        {ATTACK_AND_VALIDATE 125}
+        {SUCCEED}
+    [/event]
+)}
```

Comment:
Please add documentation for what this test is doing. Also, why is the Illuminates ability needed?

Reply:
for what alignment change was tested, Tod of test must be changed for 'lawful' alignment change for what Todbonus was diffrent of ero

Reply:
Please add a test that uses [end_turn] to make it be a lawful ToD as well then.

Reply:
i don't think what standart Tod was implemente in test files, it's why i use illuminates ability instead

Reply:
For `COMMON_KEEP_A_B_UNIT_TEST` it looks like there's already an optional TIME argument actually.

Reply:
@Pentarctagon documentation added and illuminate removed

Reply:
What I meant was that there should be both tests:
* one test that works due to Illuminates
* a second test that works because of ToD

Reply:
done